lib/omnisharp-atom/features/lib/completion-provider.ts

```diff
@@ -118,14 +118,32 @@ export var CompletionProvider = {
     excludeLowerPriority: false,
 
     makeSuggestion(item: OmniSharp.Models.AutoCompleteResponse) {
+        var description, leftLabel, iconHTML, type;
+        if atom.config.get('omnisharp-atom.useLeftLabelColumnForSuggestions') == true {
```

Comment:
I'm sure these lookups are pretty fast, but might be better to cache this value (and then use `atom.config.onDidChange()` to observe when the user changes it.